### PR TITLE
optimised day 29

### PR DESCRIPTION
**Pull request by exilednick**
The initial complexity was O(N^2) for each test case.
I optimised it to O(1) for each test case.
[https://drive.google.com/file/d/17lSvflmEsjuir19-exRoN5G9m7U25SeI/view?usp=sharing](url)

### CODE REVIEW

File: 30 days of code/Day 29- Bitwise AND/solution.cpp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#include <bits/stdc++.h>
+using namespace std;
+int main() {
+	int t,n,k;
+	cin>>t;
+	while(t--) {
+		cin>>n>>K;
```
Comment (Arejay23) [inline 30 days of code/Day 29- Bitwise AND/solution.cpp]: Varibale name not correct for K

**Review by Arejay23 — CHANGES_REQUESTED** [review]
Not able to access drive link maybe it is private.
Please check.

**Comment by exilednick** [comment]
Screenshot from 2020-10-03 12-54-45

Made the changes. The link is public but idk why not working, anyways I added the screenshot.

**Review by Arejay23 — CHANGES_REQUESTED** [review]
image




I tried to run the same code but it is failing for some sample cases please check

**Comment by exilednick** [comment]
I updated it. There was a missing bracket

**Review by Arejay23 — APPROVED** [review]

**Pull request merged**